include/ginkgo/core/log/profiler_hook.hpp

```diff
@@ -420,7 +420,6 @@ class profiling_scope_guard {
 
     profiling_scope_guard(const profiling_scope_guard&) = delete;
 
-    // TODO17: unnecessary with guaranteed RVO
```

Comment:
has this been addressed?

Reply:
I think it was necessary for some older compilers not properly implementing RVO, but I'll try removing it again